### PR TITLE
routes rating display

### PR DESCRIPTION
**Pull request by ginold**
trying to answer #243.
I'm really not sure to understand correctly the client's directives but i'm doing my best.

### CODE REVIEW

**Comment by asaunier** [comment]
> i'm doing my best.

I have absolutely no doubt about it! :)

Your changes seems to go to the right direction!

In this example http://c2corgv6-demo.gis.internal:6555/routes/54983/fr ("Aiguille Verte : Couloir Couturier"):
ratings

I think it should rather show:
```
5.4/E2 ED/S6
```
instead of
```
5.4 / E2
ED / S6
```
=> remove the blank spaces around "/" and avoid the line feed.

**Comment by ginold** [comment]
updated, ready for review.

File: c2corg_ui/templates/helpers/view.html [hunk 1]
```diff
@@ -381,3 +381,118 @@ <h3 class="heading closed show-phone" ng-click="mainCtrl.animateHeaderIcon($even
...
 </%def>
+
+
+<%def name="get_skitouring_rating(document)">\
+  % if document.get('ski_rating') or document.get('ski_exposition') or document.get('labande_ski_rating') or document.get('labande_global_rating'):
+  
+    <div class="rating-block b">
+        % if document.get('ski_rating'):
+        <span uib-tooltip="{{mainCtrl.translate('ski_rating')}}" class="value">${document['ski_rating']}</span>
+      % endif
```
Comment (asaunier) [inline c2corg_ui/templates/helpers/view.html]: ìf/endif tags are not aligned

**Comment by asaunier** [comment]
Looks good to me thanks!

**Comment by ginold** [comment]
Merge?

**Comment by asaunier** [comment]
Could you fix the if/endif alignment issue first?

**Pull request merged**